src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -252,7 +252,7 @@ CHIP_ERROR pychip_DeviceController_ConnectIP(chip::Controller::DeviceCommissione
 
 CHIP_ERROR pychip_DeviceController_DiscoverAllCommissionableNodes(chip::Controller::DeviceCommissioner * devCtrl)
 {
-    Mdns::DiscoveryFilter filter(Mdns::DiscoveryFilterType::kNone, 0);
+    Mdns::DiscoveryFilter filter(Mdns::DiscoveryFilterType::kNone, (uint16_t) 0);
```

Comment:
this seems odd - is the compiler complaining? Does it work with 0u? If not, can you do the c++ style cast?

Reply:
this was a compiler error that I get in this code